openpathsampling/high_level/transition.py

```diff
@@ -74,6 +76,7 @@ def _tps_ensemble(self, stateA, stateB):
             paths.AllInXEnsemble(stateB) & paths.LengthEnsemble(1)
         ])
 
+    # TODO: Can this kwargs be removed? It is not used
```

Comment:
@jhprinz : Were you going to remove the `kwargs` in this? As long as you add the TODO, and we agree it should go, then you might as well make the change here!